lib/API/Extra.js

```diff
@@ -644,7 +644,7 @@ module.exports = function(CLI) {
     const that = this;
     if(semver.satisfies(process.versions.node, '>= 8.0.0')) {
       this.trigger(app_name, 'internal:inspect', function (err, res) {
-        if(res[0].data.return === '') {
+        if(res && res[0].data.return === '') {
```

Comment:
shouldn't we check that the `res[0]` is not undefined ?

Reply:
Done